src/worker.h

```diff
@@ -94,6 +98,8 @@ class Worker {
   void task_info(ClientContext &context, TaskInfoRequest &request, TaskInfoReply &reply);
   // export function to workers
   bool export_function(const std::string& function);
+  // export reusable variable to workers
+  void export_reusable_variable(const std::string& name, const std::string& initializer, const std::string& reinitializer);
```

Comment:
Similarly, the code paths for shipping reusable variables and functions should be the same I think.